src/graphics/mesh.c

```diff
@@ -14,28 +44,15 @@ static void lovrMeshBindAttributes(Mesh* mesh) {
 
   VertexFormat* format = &mesh->vertexData->format;
   for (int i = 0; i < format->count; i++) {
-    Attribute attribute = format->attributes[i];
-    int location = lovrShaderGetAttributeId(shader, attribute.name);
-
-    if (location >= 0) {
-      if (mesh->enabledAttributes & (1 << i)) {
-        glEnableVertexAttribArray(location);
-
-        GLenum glType;
-        switch (attribute.type) {
-          case ATTR_FLOAT: glType = GL_FLOAT; break;
-          case ATTR_BYTE: glType = GL_UNSIGNED_BYTE; break;
-          case ATTR_INT: glType = GL_UNSIGNED_INT; break;
-        }
-
-        if (attribute.type == ATTR_INT) {
-          glVertexAttribIPointer(location, attribute.count, glType, format->stride, (void*) attribute.offset);
-        } else {
-          glVertexAttribPointer(location, attribute.count, glType, GL_TRUE, format->stride, (void*) attribute.offset);
-        }
-      } else {
-        glDisableVertexAttribArray(location);
-      }
+    lovrMeshBindAttribute(shader, mesh, format, i, mesh->enabledAttributes & (1 << i), 0);
+  }
+
+  {
+    int i; MeshAttachment attachment;
+    vec_foreach(&mesh->attachments, attachment, i) {
+      lovrGraphicsBindVertexBuffer(attachment.mesh->vbo);
+      // TODO: Allow disabling of attached attributes?
```

Comment:
I wonder if it should be inherited from the other Mesh's state?  like `bool enabled = lovrMeshIsAttributeEnabled(attachment.mesh, attributeName)`